2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Configure stats pid to 2987264
2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Loading settings from /home/geng/.config/wandb/settings
2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Loading settings from /home/geng/WARDEN/src/wandb/settings
2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'src/multi_watermark_run_gpt_backdoor.py', 'program_abspath': '/home/geng/WARDEN/src/multi_watermark_run_gpt_backdoor.py', 'program': 'multi_watermark_run_gpt_backdoor.py'}
2024-11-01 19:36:09,881 INFO    MainThread:2987264 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-11-01 19:36:09,882 INFO    MainThread:2987264 [wandb_init.py:_log_setup():529] Logging user logs to /home/geng/WARDEN/src/wandb/run-20241101_193609-j8rq3bv0/logs/debug.log
2024-11-01 19:36:09,882 INFO    MainThread:2987264 [wandb_init.py:_log_setup():530] Logging internal logs to /home/geng/WARDEN/src/wandb/run-20241101_193609-j8rq3bv0/logs/debug-internal.log
2024-11-01 19:36:09,882 INFO    MainThread:2987264 [wandb_init.py:init():569] calling init triggers
2024-11-01 19:36:09,882 INFO    MainThread:2987264 [wandb_init.py:init():576] wandb.init called with sweep_config: {}
config: {}
2024-11-01 19:36:09,882 INFO    MainThread:2987264 [wandb_init.py:init():619] starting backend
2024-11-01 19:36:09,882 INFO    MainThread:2987264 [wandb_init.py:init():623] setting up manager
2024-11-01 19:36:09,885 INFO    MainThread:2987264 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-11-01 19:36:09,887 INFO    MainThread:2987264 [wandb_init.py:init():631] backend started and connected
2024-11-01 19:36:09,895 INFO    MainThread:2987264 [wandb_init.py:init():720] updated telemetry
2024-11-01 19:36:09,986 INFO    MainThread:2987264 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-11-01 19:37:40,004 ERROR   MainThread:2987264 [wandb_init.py:init():779] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-11-01 19:37:40,004 ERROR   MainThread:2987264 [wandb_init.py:init():1191] error in wandb.init()
Traceback (most recent call last):
  File "/home/geng/miniconda3/envs/EmbMarker/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    return wi.init()
  File "/home/geng/miniconda3/envs/EmbMarker/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 785, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-11-01 19:37:43,017 WARNING MsgRouterThr:2987264 [router.py:message_loop():77] message_loop has been closed
